src/DDTrace/Tags.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace DDTrace\Tags;
+
+const ENV = 'env';
+const SPAN_TYPE = 'span.type';
+const SERVICE_NAME = 'service.name';
```

Comment:
What does service and resource name map to on the DataDog side of things?

Reply:
A service is a particular webapp, or maybe microservice, or a database. A resource is a particular query handled by that resource, like a path (/myaccount) or a query (select params from users where id = ?). See https://help.datadoghq.com/hc/en-us/articles/115000702546-What-is-the-Difference-Between-Type-Service-Resource-and-Name- for more.